randovania/games/prime1/json_data/Tallon Overworld.txt

```diff
@@ -551,9 +551,13 @@ Extra - aabb: [-703.2118, 297.52924, -49.04736, -619.3062, 378.77765, 64.30938]
       Trivial
   > Pickup (Missile Expansion)
       All of the following:
-          # https://youtu.be/x2zjlvSy0Og
           Space Jump Boots
-          Combat/Scan Dash (Intermediate) or Use Grapple Beam
+          Any of the following:
+              # https://youtu.be/x2zjlvSy0Og
+              Combat/Scan Dash (Intermediate) or Use Grapple Beam
+              All of the following:
+                  # https://youtu.be/3sHPzMRqmII
+                  R-Jump (Intermediate) and Standable Terrain (Intermediate)
```

Comment:
Nitpick, but this looks like it might be possible with just an L-Jump. If you haven't investigated that yet, please do.

Reply:
This is possible with an L-Jump. <https://youtu.be/i2J6z6wrLGw> 
Found it to be fairly precise but can be made easier by using the initial rock as a slope.

Reply:
`Slope Jump (Intermediate) and L-Jump (Intermediate) and Standable Terrain (Intermediate)`

In the video, you actually do a slope jump.

Reply:
I see the sloped movement now, will correct the video description in line with this. Logic changed as recommended.